schemas/ingest-events.v1.schema.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Error message from Relay",
+  "type": "object",
+  "properties": {
+    "type": { "type": "string" },
+    "payload": {
+      "description": "bytes"
+    },
```

Comment:
JSON schema allows additionalProperties by default, right? Because Relay does send more fields: https://github.com/getsentry/relay/blob/87b23e6953cd18a0d591c9d13e43baee0d0d4b95/relay-server/src/services/store.rs#L1268-L1271

Reply:
It does -- the example included here also has those extra fields and is passing the tests